### PR TITLE
Devtools week che dogfooding test development

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
* Modifies existing devfile to allow for development of Che/DevSpaces tests on Che Dogfooding instance

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/22191

### How to test this PR?

* Apply this fork as a factory to dogfooding and try to run the described command

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .devfile.yaml [hunk 1]
```diff
@@ -9,16 +9,29 @@
...
+  language: typescript
 attributes:
-  controller.devfile.io/storage-type: ephemeral
+  controller.devfile.io/scc: container-build
+  controller.devfile.io/storage-type: per-workspace
+projects:
+  - name: che
+    git:
+      remotes:
+        origin: 'https://github.com/ScrewTSW/che.git'
```
Comment (nallikaea) [inline .devfile.yaml]: personal repo

**Comment by nallikaea** [comment]
@ScrewTSW which branch do you use to develop this issue? Looks it a long time outdated 
image

File: .devfile.yaml (hunk 1 above)
Comment (ScrewTSW) [reply .devfile.yaml]: yes, for testing purposes until draft is removed and to correctly pull the updated devfile.

**Comment by ScrewTSW** [comment]
> @ScrewTSW which branch do you use to develop this issue? Looks it a long time outdated

yes, that's true. I've forked it quite some time ago and did not rebase in a while.

File: .devfile.yaml (hunk 1 above)
Comment (ScrewTSW) [reply .devfile.yaml]: removed.

**Comment by ScrewTSW** [comment]
> @ScrewTSW which branch do you use to develop this issue? Looks it a long time outdated image

updated

**Review by dmytro-ndp — APPROVED** [review]
Looks good to merge (assuming that PR has nothing with `update-contributing` devfile command).

**Comment by ibuziuk** [comment]
@dmytro-ndp this looks like useful PR
maybe you can take over it in the next sprint update & merge?

**Comment by ibuziuk** [comment]
@musienko-maxim please, update the PR so that we can merge it

**Pull request merged**